src/Apps/Order/Routes/Shipping2/Components/FulfillmentDetailsForm.tsx

```diff
@@ -492,17 +555,16 @@ const FulfillmentDetailsFormLayout = (
               data-testid="FulfillmentDetailsForm_saveAddress"
               tabIndex={tabbableIf("new_address")}
               onSelect={selected => {
-                setFieldValue("attributes.saveAddress", selected)
+                setFieldValue("meta.saveAddress", selected)
```

Comment:
Moved non-address values into a special `meta` object. This made a lot of other mutation-arg-passing cleaner.

Reply:
makes sense 👍